migration_tools.drush.inc

```diff
@@ -248,6 +263,66 @@ function drush_migration_tools_mt_migrate_file_folders($organization) {
 }
 
 /**
+ * Drush command callback for migration-tools-html-file-list.
+ *
+ */
+function drush_migration_tools_html_file_list($directory) {
+  // Set up options
+  $format = drush_get_option('format', 'txt');
+  $outfile = drush_get_option('outfile');
+  $mask = '/.*\.htm(l)?$/i';
+
+  switch ($format) {
+    case "csv":
+      $delimiter = ", ";
+      $newline = "\n";
+      break;
+
+    default:
+      $delimiter = "\t";
+      $newline = "\n";
+  }
+
+  // Fetch file list
+  $options = array();
+  $files = file_scan_directory($directory, $mask, $options);
+
+  // Generate output
+  $output = "Path" . $delimiter . $newline;
+  foreach ($files as $file) {
+    $output .= $file->uri . $delimiter;
+    $output .= $newline;
+  }
+
+  // Attempt to write results to file if requested, otherwise output to terminal.
+  if ($outfile) {
```

Comment:
Outfile seems to not really be used now and $format seems to always be defined, unless intentionally set to empty quotes.  Perhaps this check needs to become

````
if (empty($format) || $format === 'screen' )
````